src/script/microbit-interfacing/MicrobitSerial.ts

```diff
@@ -94,20 +110,19 @@ export class MicrobitSerial implements MicrobitConnection {
       await this.handshake();
       stateOnConnected(DeviceRequestStates.INPUT);
 
-      // Check for USB being unplugged
-      navigator.usb.addEventListener('disconnect', () => {
-        logMessage('USB disconnected');
-        this.stopConnectionCheck();
-      });
```

Comment:
The handleError callback takes care of this case so we don't need a separate listener now.